src/main/webapp/WEB-INF/view/searchResults.jsp

```diff
@@ -1,5 +1,6 @@
 <%@page pageEncoding="UTF-8" contentType="text/html; charset=UTF-8"%>
 <%-- query matched some document --%>
+<script type="text/javascript" src="${fe:url('/js/clipboard.min.js')}"></script>
```

Comment:
Please move this line to the bottom of search.jsp.

Reply:
Thank you for pointing this out.
I have moved it to the bottom of search.jsp.